apps/server/src/shared/domain/entity/lesson.entity.ts

```diff
@@ -9,6 +9,19 @@ export interface ILessonProperties {
 	hidden?: boolean;
 	course: Course;
 	position?: number;
+	contents: IComponentProperties[] | [];
+}
+
+export interface IContentProperties {
+	content: {
+		text: string;
```

Comment:
this only works for text content, which is ok for current goal
but it would be nice to come up with a pattern, so that we can expand for the other types of content too